internal/controller/pathwaysjob_controller.go

```diff
@@ -471,19 +471,34 @@ func MakeSuccessPolicy(pw *pathwaysjob.PathwaysJob) *jobsetv1alpha2.SuccessPolic
 	}
 }
 
+// Pick the RM Image based on whether custom image or Pathways version are provided.
+func MakeResourceManagerAndWorkerImage(pw *pathwaysjob.PathwaysJob) string {
+	if pw.Spec.CustomComponents != nil && pw.Spec.CustomComponents.PathwaysServerImage != "" {
+		return pw.Spec.CustomComponents.PathwaysServerImage
+	} else {
+		return fmt.Sprintf("us-docker.pkg.dev/cloud-tpu-v2-images/pathways/server:%s", makeImageTagUsingPathwaysVersion(pw))
```

Comment:
Maybe refactor the prefix to a constant? A few other similar situations for other images. Feel free to skip if I'm missing context.

Reply:
Great suggestion, done!